awx/main/migrations/0100_v370_unifiedjob_canceled.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.4 on 2019-11-25 20:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0099_v370_projectupdate_job_tags'),
```

Comment:
This file probably needs to be `0101_v370`, we've added a `0100` at this point.

Reply:
@thedoubl3j drop me a line and I can link you to my git rebase script that will renumber migrations for you.

Reply:
+1 @jbradberry definitely. plan on picking back up the work on this either tomorrow or thursday